.changeset/good-turkeys-chew.md

```diff
@@ -0,0 +1,18 @@
+---
+'@fuels/react': minor
+---
+
+- Add `useTransactionResult` hook to get a transaction that has been successfully mined.
```

Comment:
```suggestion
- Add `useTransactionResult` hook to get a transaction that has been executed
```

Reply:
not necessarily will be "successfully", it can be error as well

Reply:
makes sense.

https://github.com/FuelLabs/fuels-npm-packs/pull/94/commits/2ad2c8e7a0f053fa1846ce326731ccc75708f1d1